samples/csharp/getting-started/Ranking_PersonalizedSort/README.md

```diff
@@ -0,0 +1,215 @@
+# Rank hotel search results to provide personalized sorting
+
+| ML.NET version | API type          | Status                        | App Type    | Data type | Scenario            | ML Task                   | Algorithms                  |
+|----------------|-------------------|-------------------------------|-------------|-----------|---------------------|---------------------------|-----------------------------|
+| v1.1.0         | Dynamic API       | Up-to-date                    | Console app | .csv file | Ranking hotel search results | Ranking          | LightGbm |
+
+This introductory sample shows how to use ML.NET to predict the relevance and order of hotel search results. In the world of machine learning, this type of prediction is known as ranking.
```

Comment:
```suggestion
This introductory sample shows how to use ML.NET to predict the best order to display hotel search results. In the world of machine learning, this type of prediction is known as ranking.
```

Reply:
I'm unsure if it's considered a bug or design choice at this point. I find `{"Perfect", "Excellent", "Good", "Fair", or "Bad"}` to be more user friendly. But the design of the API isn't my area.

For your exact error, it may be that you have to change your Label column from `Int` to `String`.